src/main/java/slimeknights/mantle/command/MantleCommand.java

```diff
@@ -32,7 +32,10 @@ public class MantleCommand {
   public static SuggestionProvider<CommandSourceStack> VALID_TAGS;
   /** Suggestion provider that lists tags values for this registry */
   public static SuggestionProvider<CommandSourceStack> REGISTRY_VALUES;
-  /** Suggestion provider that lists registered book ids **/
+  /**
+   * Suggestion provider that lists registered book ids
+   * @deprecated Use {@link slimeknights.mantle.client.book.BookLoader#getRegisteredBooks()} instead
```

Comment:
Shouldn't this be `MantleClientCommand#REGISTERED_BOOKS`?

Reply:
Whoops, yes, case of copy-paste mishap